src/test/components/ProfileViewerContainer.test.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import { shallowWithStore } from '../fixtures/enzyme';
+
+import ProfileViewerContainer from '../../components/app/ProfileViewerContainer';
+import { changeSelectedTab } from '../../actions/app';
+import { storeWithProfile } from '../fixtures/stores';
+import { getProfileFromTextSamples } from '../fixtures/profiles/make-profile';
+
+import type { TabSlug } from '../../types/actions';
+
+describe('app/ProfileViewerContainer', function() {
+  const { profile } = getProfileFromTextSamples(`
```

Comment:
I know I wrote it, so I probably should say anything, but I really do like this function 🤓